Frontend/ui-driver/src/Helpers/Utils.purs

```diff
@@ -1082,4 +1082,10 @@ getVehicleVariantName variant =
                   CarCategory -> getString CAB
                   BikeCategory -> getString BIKE_TAXI
                   AmbulanceCategory -> getString AMBULANCE
-                  UnKnown -> ""
\ No newline at end of file
+                  UnKnown -> ""
+
+dummyNotificationBody :: NotificationBody
+dummyNotificationBody = {
+  title : "",
+  message : ""
```

Comment:
can you give a proper name here? like `initialNotificationBody` or `defaultNotificationBody`